android/build.gradle

```diff
@@ -1,38 +1,22 @@
 
-buildscript {
-    repositories {
-        jcenter()
-    }
+apply plugin: 'com.android.library'
 
-    dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
-    }
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-
-apply plugin: 'com.android.library'
-
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.1"
+    compileSdkVersion safeExtGet('compileSdkVersion', 25)
+    buildToolsVersion safeExtGet('buildToolsVersion', "25.0.1")
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 22)
         versionCode 1
         versionName "1.0"
-        ndk {
-            abiFilters "armeabi-v7a", "x86"
-        }
     }
 }
 
-repositories {
-    mavenCentral()
-}
-
-
 dependencies {
-    compile 'com.tom_roush:pdfbox-android:1.8.10.0'
```

Comment:
Just tried this fork - it fixed the original error but produced a new error:
```
project/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java:25: error: package com.tom_roush.pdfbox.pdmodel does not exist
import com.tom_roush.pdfbox.pdmodel.PDDocument;
project/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java:80: error: cannot find symbol
PDDocument myDocument = PDDocument.load(mPdfFile);
                                ^
symbol: class PDDocument
```

This seems like it could possibly be caused by removing this dependency (`com.tom_roush:pdfbox-android:1.8.10.0`)?

Reply:
If I'm not wrong then this issue is probably happening due to inaccurate linking native libraries. Please a fresh setup then should work or link all of them manually maybe.

Thanks